lib/reply.js

```diff
@@ -526,6 +527,9 @@ function handleError (reply, error, cb) {
     if (result && typeof result.then === 'function') {
       wrapThenable(result, reply)
```

Comment:
Note that if result is a thenable, it might reject. If that happens, we would need to do the same.